app/models/current.rb

```diff
@@ -0,0 +1,3 @@
+class Current < ActiveSupport::CurrentAttributes
```

Comment:
Pop quiz for you. What is this for? And why is this controversial when overused?